### PR TITLE
Ghosts can now jump to emotes, better follow links, ghosts can now see where someone ghosted.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
As per title.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/mob/mob_helpers.dm [hunk 1]
```diff
@@ -580,11 +580,11 @@ proc/is_blind(A)
 /proc/announce_ghost_joinleave(O, var/joined_ghosts = 1, var/message = "")
 	var/client/C
 	//Accept any type, sort what we want here
-	if(istype(O, /mob))
+	if(ismob(O)))
```
Comment (Doxxmedearly) [inline code/modules/mob/mob_helpers.dm:583]:
```suggestion
	if(ismob(O))
```

File: code/modules/mob/emote.dm [hunk 2]
```diff
@@ -56,7 +56,7 @@
 /mob/proc/send_emote(var/message, var/type)
 	var/list/messageturfs = list()//List of turfs we broadcast to.
 	var/list/messagemobs = list()//List of living mobs nearby who can hear it, and distant ghosts who've chosen to hear it
```
Comment (Arrow768) [inline code/modules/mob/emote.dm:58]: Update the comment

Comment (skull132) [reply code/modules/mob/emote.dm:58]: Delete the comment -- comments lie.

**Pull request merged**